code/src/components/NewTodo.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import todo from 'reducers/todo'
+import uniqid from 'uniqid'
+import styled from 'styled-components'
+import { Section } from './styling/Wrappers'
+
+const NewTodo = () => {
+  const dispatch = useDispatch()
+  const [newTodo, setNewTodo] = useState('')
+
+  const onFormSubmit = (event) => {
+    event.preventDefault()
+    const postNewTodo = {
+      id: uniqid(),
+      name: newTodo,
+      isCompleted: false
+    }
+    dispatch(todo.actions.addItem(postNewTodo))
+    setNewTodo('')
+  }
+
+  return (
+    <Section>
+      <form onSubmit={onFormSubmit}>
```

Comment:
A suggestion is to add "disabled={inputValue.length < 2 || inputValue.length > 25}" or similar to restrict the 0/too long input.